javascript.js

```diff
@@ -63,8 +63,22 @@ const switchButton = document.querySelector('.button__bookmark')
 // })
 
 /* Switch bookmark Button per if..else method) */
-switchButton.addEventListener('click', () => {
-  if (switchButton.classList.contains('button__bookmark--clicked')) {
-    switchButton.classList.remove('button__bookmark--clicked')
-  } else switchButton.classList.add('button__bookmark--clicked')
+// switchButton.addEventListener('click', () => {
+//   if (switchButton.classList.contains('button__bookmark--clicked')) {
+//     switchButton.classList.remove('button__bookmark--clicked')
+//   } else switchButton.classList.add('button__bookmark--clicked')
+// })
+
+/* Switch bookmark Button für alle bookmarks */
+
+const switchButtonArray = document.querySelectorAll('.button__bookmark')
+
+switchButtonArray.forEach(switchButton => {
+  switchButton.addEventListener('click', () => {
+    if (switchButton.classList.contains('button__bookmark--clicked')) {
+      switchButton.classList.remove('button__bookmark--clicked')
+    } else {
+      switchButton.classList.add('button__bookmark--clicked')
+    }
```

Comment:
How about `switchButton.classList.toggle('button__bookmark--clicked')`? 🙂

Reply:
Yes, that's the shorter version of the if...else statement? I think, I used it, when I wrote the code in the early beginning. But I also wanted to see, if I would figure out the if...else version ^^

Reply:
Ah I see! 🙂